### PR TITLE
create tests for early boot

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**
Signed-off-by: pbanasiak <przemyslaw.banasiak@3mdeb.com>

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md [hunk 1]
```diff
@@ -16,29 +32,27 @@ The test case verifies the DMA protection has been configured properly.
...
+    [Test cases common documentation](#test-cases-common-documentation) section.
 
 **Test steps**
 
 1. Power on the DUT.
-2. Boot into the system.
-3. Log into the system by using the proper login and password.
-4. Open a terminal window and execute the following command:
+1. While booting, press the `SETUP_MENU_KEY` to enter Setup Menu.
+1. Enter the `Secure Boot Configuration` submenu.
```
Comment (Pokisiekk) [inline docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md:41]:
```suggestion
1. While booting, press the `SETUP_MENU_KEY` to enter Setup Menu.
1. Enter the `Device Manager` menu using the arrow keys and Enter.
1. Enter the `Secure Boot Configuration` submenu.
```

File: docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md [hunk 2]
```diff
@@ -93,3 +107,55 @@ Pmr(1) disabled
...
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. While booting, press the `SETUP_MENU_KEY` to enter Setup Menu.
+1. Enter the `Secure Boot Configuration` submenu.
```
Comment (Pokisiekk) [inline docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md:133]:
```suggestion
1. While booting, press the `SETUP_MENU_KEY` to enter Setup Menu.
1. Enter the `Device Manager` menu using the arrow keys and Enter.
1. Enter the `Secure Boot Configuration` submenu.
```

**Review by Pokisiekk — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md (hunk 1 above)
Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md]: @Pokisiekk path has been updated

File: docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md]: @Pokisiekk path has been updated

**Review by Pokisiekk — APPROVED** [review]

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**